lib/screens/v2/widget_instance/alert.ex

```diff
@@ -358,6 +365,11 @@ defmodule Screens.V2.WidgetInstance.Alert do
 
   # For GL, we want to list all affected branches for the alert and not just the branch serving the home stop.
   # This allows us to show a pill for each branch in the informed_entities of the alert (or GL pill if all branches are affected).
+  
+  # This is a special exception to how informed_routes are generated for other screens.
+  # I'm not sure that these should be two clauses of the same function, because they aim to return different kinds of values
+  # One is informed routes at the home stop (seen in BaseAlert also)
+  # The other is informed routes for the alert
```

Comment:
I'm not following in your explanation of the bug in the GL clause. Could you describe differently? It seems to be outputting what we'd expect -- two branches if there are only two branches, "Green" if more than two.

Reply:
I think I will generalize the `get_affected_routes` function from `reconstructed_alert.ex`, move it to replace this function in BaseAlert, and will write it in a way that it will be more clear what it is doing.